src/main/frontend/handler/page.cljs

```diff
@@ -163,7 +163,7 @@
           (db/transact! [(assoc page-id :block/properties {key value})])
           (db/refresh! repo {:key :block/change
                              :data [block]})
-          ;; (ui-handler/re-render-root!)
+          (ui-handler/re-render-root!)
```

Comment:
@tiensonqin have to uncomment this so title property get properly changed, not sure if this is intended?

Reply:
Actually, I have no idea 😂  Testing this PR now.